### ISSUE TITLE
add chrome extension

### ISSUE THREAD

**Issue by dsernst** [cameron/squirt#35]
For those of us that don't want to show a bookmark bar, could this be supported in a Chrome Extension? Either a toolbar button or option in the context menu?

Such as https://chrome.google.com/webstore/detail/spread-speed-reading-exte/hpnjoanfahcolkdgnfecmncnlfklnllk?hl=en

**Comment by cameron** [comment]
I would love to add an extension!

**Comment by PatrickJS** [comment]
+1

**Comment by luczyna** [comment]
I also think this would be great!

**Comment by nrdsrfr** [comment]
I converted to an extension here http://sandbox.self.li/bookmarklet-to-extension/

**Comment by LucasZapico** [comment]
agreed. @nrdsrfr solid I'm going to check out how you did it. Thanks for the link.
